src/wallet/wallet.h

```diff
@@ -656,6 +657,9 @@ class CWallet final : public CCryptoKeyStore, public CValidationInterface
     static std::atomic<bool> fFlushScheduled;
     std::atomic<bool> fAbortRescan;
     std::atomic<bool> fScanningWallet;
```

Comment:
Add a comment saying this is controlled by `WalletRescanReserver`.

Reply:
It's not really controlled by `WalletRescanReserver`. This was added before and `ScanForWalletTransactions` sets it to `true`.